### ISSUE TITLE
Recent changes in GAP `master` break the tests

### ISSUE THREAD

**Issue by james-d-mitchell** [semigroups/Semigroups#905]
As per the title I get the following, I think these are harmless (caused by a new method for `IsomorphismPermGroup` which returns a lower degree representation). 

See also:

https://github.com/gap-system/gap/issues/5318

~~~
gap> SemigroupsTestStandard();
#I  Running tests with acting methods enabled
Architecture: aarch64-apple-darwin21-default64-kv8

testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/acting.tst
     153 ms (117 ms GC) and 48.2MB allocated for attributes/acting.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/attr.tst
     702 ms (110 ms GC) and 400MB allocated for attributes/attr.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/dual.tst
     213 ms (88 ms GC) and 50.3MB allocated for attributes/dual.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/factor.tst
     607 ms (102 ms GC) and 242MB allocated for attributes/factor.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/homomorph.tst
     833 ms (133 ms GC) and 359MB allocated for attributes/homomorph.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/inverse.tst
     183 ms (96 ms GC) and 83.3MB allocated for attributes/inverse.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/isomorph.tst
     565 ms (101 ms GC) and 108MB allocated for attributes/isomorph.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/isorms.tst
    2088 ms (206 ms GC) and 1.36GB allocated for attributes/isorms.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/maximal.tst
     334 ms (111 ms GC) and 222MB allocated for attributes/maximal.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/properties.tst
######## > Diff in:
/Users/jdm/gap/pkg/semigroups/tst/standard/attributes/properties.tst:2001
# Input is:
H := InverseSemigroup(G.1, rec(acting := true));
# Expected output:
<partial perm group of rank 8 with 1 generator>
# But found:
<partial perm group of rank 4 with 1 generator>
########
     639 ms (130 ms GC) and 486MB allocated for attributes/properties.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/semifp.tst
     104 ms (102 ms GC) and 498KB allocated for attributes/semifp.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/attributes/translat.tst
######## > Diff in:
/Users/jdm/gap/pkg/semigroups/tst/standard/attributes/translat.tst:437
# Input is:
l := LeftTranslation(L,
[(1, 6, 3, 9)(2, 7, 5, 4)(8, 10, 11, 12),
(1, 6, 3, 9)(2, 7, 5, 4)(8, 10, 11, 12)],
Transformation([2, 2]));
# Expected output:
<left translation on <simple semigroup of size 96, with 6 generators>>
# But found:
Error, the second argument must be a list of group elements of length equal to\
 the number of rows of the underlying semigroup of the first argument
########
######## > Diff in:
/Users/jdm/gap/pkg/semigroups/tst/standard/attributes/translat.tst:442
# Input is:
r := RightTranslation(R,
[(1, 2, 3, 5)(4, 10, 7, 12)(6, 11, 9, 8), (), (), ()],
Transformation([2, 3, 4, 1]));
# Expected output:
<right translation on <simple semigroup of size 96, with 6 generators>>
# But found:
Error, the second argument must be a list of group elements of length equal to\
 the number of rows of the underlying semigroup of the first argument
########
######## > Diff in:
/Users/jdm/gap/pkg/semigroups/tst/standard/attributes/translat.tst:449
# Input is:
l = l2 and r = r2;
# Expected output:
true
# But found:
false
########
######## > Diff in:
/Users/jdm/gap/pkg/semigroups/tst/standard/attributes/translat.tst:751
# Input is:
Bitranslation(H, (1, 2, 3, 5)(4, 10, 7, 12)(6, 11, 9, 8),
Transformation([2, 2]), Transformation([2, 3, 2]));
# Expected output:
Error, the arguments given do not define a bitranslation
# But found:
Error, the second argument must be an element of the group that the underlying\
 semigroup of the first argument is defined over
########
    1174 ms (154 ms GC) and 419MB allocated for attributes/translat.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/cong.tst
     196 ms (119 ms GC) and 113MB allocated for congruences/cong.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/conginv.tst
     227 ms (115 ms GC) and 171MB allocated for congruences/conginv.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/conglatt.tst
     606 ms (140 ms GC) and 166MB allocated for congruences/conglatt.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/congpairs.tst
     270 ms (125 ms GC) and 32.4MB allocated for congruences/congpairs.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/congrees.tst
     190 ms (117 ms GC) and 139MB allocated for congruences/congrees.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/congrms.tst
     159 ms (114 ms GC) and 22.9MB allocated for congruences/congrms.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/congsemigraph.tst
     192 ms (115 ms GC) and 21.1MB allocated for congruences/congsemigraph.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/congsimple.tst
     155 ms (115 ms GC) and 32.5MB allocated for congruences/congsimple.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/conguniv.tst
     133 ms (114 ms GC) and 22.8MB allocated for congruences/conguniv.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/congruences/congwordgraph.tst
     120 ms (112 ms GC) and 1.94MB allocated for congruences/congwordgraph.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/bipart.tst
     176 ms (128 ms GC) and 14.6MB allocated for elements/bipart.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/blocks.tst
     124 ms (120 ms GC) and 2.12MB allocated for elements/blocks.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/boolmat.tst
     129 ms (119 ms GC) and 3.18MB allocated for elements/boolmat.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/elements.tst
     125 ms (122 ms GC) and 1.78MB allocated for elements/elements.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/ffmat.tst
     161 ms (122 ms GC) and 20.1MB allocated for elements/ffmat.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/maxplusmat.tst
     130 ms (118 ms GC) and 3.27MB allocated for elements/maxplusmat.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/pbr.tst
     123 ms (117 ms GC) and 2.03MB allocated for elements/pbr.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/pperm.tst
     112 ms (112 ms GC) and 85.6KB allocated for elements/pperm.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/semiringmat.tst
     123 ms (113 ms GC) and 3.26MB allocated for elements/semiringmat.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/elements/trans.tst
     317 ms (116 ms GC) and 50.1MB allocated for elements/trans.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/fp/freeband.tst
     216 ms (115 ms GC) and 40.4MB allocated for fp/freeband.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/fp/freeinverse.tst
     312 ms (119 ms GC) and 67.4MB allocated for fp/freeinverse.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/fp/tietze.tst
     233 ms (115 ms GC) and 18.8MB allocated for fp/tietze.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/fp/word.tst
     113 ms (113 ms GC) and 170KB allocated for fp/word.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/greens/acting-inverse.tst
     136 ms (114 ms GC) and 16.4MB allocated for greens/acting-inverse.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/greens/acting-regular.tst
     138 ms (114 ms GC) and 35.2MB allocated for greens/acting-regular.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/greens/acting.tst
     739 ms (143 ms GC) and 331MB allocated for greens/acting.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/greens/froidure-pin.tst
     363 ms (139 ms GC) and 251MB allocated for greens/froidure-pin.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/greens/generic.tst
     302 ms (134 ms GC) and 194MB allocated for greens/generic.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/ideals/acting.tst
     200 ms (125 ms GC) and 128MB allocated for ideals/acting.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/ideals/froidure-pin.tst
     124 ms (115 ms GC) and 9.25MB allocated for ideals/froidure-pin.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/ideals/ideals.tst
     646 ms (182 ms GC) and 665MB allocated for ideals/ideals.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/libsemigroups/cong.tst
     166 ms (140 ms GC) and 22.4MB allocated for libsemigroups/cong.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/libsemigroups/froidure-pin.tst
     192 ms (140 ms GC) and 21.1MB allocated for libsemigroups/froidure-pin.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/libsemigroups/sims1.tst
     430 ms (130 ms GC) and 43.7MB allocated for libsemigroups/sims1.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/main/acting.tst
     206 ms (121 ms GC) and 85.9MB allocated for main/acting.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/main/froidure-pin.tst
     203 ms (123 ms GC) and 38.8MB allocated for main/froidure-pin.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/main/semiact.tst
     234 ms (122 ms GC) and 68.6MB allocated for main/semiact.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/main/setup.tst
     157 ms (120 ms GC) and 17.0MB allocated for main/setup.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/obsolete.tst
     118 ms (118 ms GC) and 38.4KB allocated for obsolete.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/options.tst
     117 ms (117 ms GC) and 134KB allocated for options.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/grpperm.tst
     150 ms (124 ms GC) and 11.0MB allocated for semigroups/grpperm.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semibipart.tst
     457 ms (136 ms GC) and 179MB allocated for semigroups/semibipart.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semiboolmat.tst
     394 ms (128 ms GC) and 95.9MB allocated for semigroups/semiboolmat.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semicons.tst
     887 ms (140 ms GC) and 161MB allocated for semigroups/semicons.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semidp.tst
     611 ms (162 ms GC) and 238MB allocated for semigroups/semidp.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semieunit.tst
     335 ms (133 ms GC) and 118MB allocated for semigroups/semieunit.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semiex.tst
     508 ms (143 ms GC) and 187MB allocated for semigroups/semiex.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semiffmat.tst
    1136 ms (167 ms GC) and 405MB allocated for semigroups/semiffmat.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semifp.tst
     537 ms (149 ms GC) and 161MB allocated for semigroups/semifp.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semigraph.tst
     139 ms (137 ms GC) and 767KB allocated for semigroups/semigraph.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semigrp.tst
     902 ms (183 ms GC) and 513MB allocated for semigroups/semigrp.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semimaxplus.tst
     308 ms (160 ms GC) and 38.1MB allocated for semigroups/semimaxplus.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semipbr.tst
     368 ms (157 ms GC) and 87.9MB allocated for semigroups/semipbr.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semipperm.tst
######## > Diff in:
/Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semipperm.tst:1787
# Input is:
T := AsSemigroup(IsPartialPermSemigroup, S);
# Expected output:
<partial perm group of size 6, rank 6 with 2 generators>
# But found:
<partial perm group of size 6, rank 3 with 2 generators>
########
######## > Diff in:
/Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semipperm.tst:1958
# Input is:
Range(map);
# Expected output:
<partial perm group of size 8, rank 8 with 3 generators>
# But found:
<partial perm group of size 8, rank 4 with 3 generators>
########
     576 ms (160 ms GC) and 251MB allocated for semigroups/semipperm.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semiquo.tst
     275 ms (148 ms GC) and 72.3MB allocated for semigroups/semiquo.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semirms.tst
    2821 ms (549 ms GC) and 5.89GB allocated for semigroups/semirms.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/semigroups/semitrans.tst
     899 ms (188 ms GC) and 428MB allocated for semigroups/semitrans.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/tools/display.tst
     246 ms (152 ms GC) and 28.1MB allocated for tools/display.tst
testing: /Users/jdm/gap/pkg/semigroups/tst/standard/tools/io.tst
     177 ms (148 ms GC) and 7.57MB allocated for tools/io.tst
-----------------------------------
total     28434 ms (9746 ms GC) and 15.3GB allocated
              7 failures in 3 of 72 files

#I  Errors detected while testing

false
~~~

**Comment by fingolfin** [comment]
Now that a fix is merged, it would be fantastic if a new release could be made (perhaps after merging PR #907)

**Comment by james-d-mitchell** [comment]
Sure I can make a release tomorrow or Monday, thanks for the fixes

**Comment by james-d-mitchell** [comment]
Resolved in v5.2.1
